MANIFEST.in

```diff
@@ -1,2 +1 @@
-include metalearn/metafeatures/metafeatures.json
 include metalearn/metafeatures/metafeatures_schema.json
```

Comment:
FWIW, PEP and Linux standards are to have a newline at the end of the file

Reply:
YOIWAL (Your opinion is worth a lot). Thanks for the info. I knew it was PEP standard but didn't know it was linux too. If we want, maybe we could add that in another issue.